windows_containers/index.adoc

```diff
@@ -0,0 +1,27 @@
+[id="windows-container-overview"]
+= Windows Container Support for OpenShift overview
+include::modules/common-attributes.adoc[]
+:context: windows-container-overview
+
+toc::[]
+
+Windows Container Support for OpenShift is a feature providing the ability to run Windows compute nodes in an {product-title} cluster. This is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With a Red Hat subscription, you can get support for running Windows workloads in {product-title}. For more information, see the xref:../windows_containers/windows-containers-release-notes-4-x.adoc#windows-containers-release-notes-4-x[release notes].
+
+Windows container workloads: For workloads including both Linux and Windows, {product-title} allows you to deploy Windows workloads running on Windows Server containers while also providing traditional Linux workloads hosted on {op-system-first} or {op-system-base-full}.
+
+For more information see, xref:../windows_containers/understanding-windows-container-workloads.adoc#understanding-windows-container-workloads[getting started with Windows container workloads].
+
+Enable Windows container workloads: You need a WMCO to run Windows workloads in your cluster. The WMCO orchestrates the process of deploying and managing Windows workloads on a cluster. For more information see, xref:../windows_containers/enabling-windows-container-workloads.adoc#enabling-windows-container-workloads[how to enable Windows container workloads].
+
+Creating a Windows `MachineSet` object: You can create a Windows `MachineSet` object to create infrastructure Windows machine sets and related machines so that you can move supporting Windows workloads to the new Windows machines. You can create a Windows `MachineSet` object on the multiple platforms.
```

Comment:
Most lgtm, here I see we mentioned the way of creating Windows worker node by MachineSet, using Bring-Your-Own-Host (BYOH) Windows instances as nodes is also supported, should we add it to the overview too?

Reply:
I haven't seen the doc mentioning it. So if we add it here, we won't have a section to point to. The customer would need the information in detail. I might be wrong, do we have such a section? @sgaoshang

Reply:
We have that section, pls take a look here https://deploy-preview-40173--osdocs.netlify.app/openshift-enterprise/latest/windows_containers/byoh-windows-instance.html

Reply:
That's cool. I will add this info to this overview page

Reply:
@sgaoshang thank you for the suggestion. I added the BYOH part.